layouts/index.html

```diff
@@ -340,16 +347,26 @@ <h2>Open Source</h2>
             <p>RAPIDS would not be possible without the collaboration of these important open source projects. Click on a logo to
               learn more: </p>
             <div class="d-flex flex-wrap justify-content-between mt-5">
-              {{ $row := slice "apache-arrow" "dask" "nuclio" "numba" "scikit-learn" "xgboost" }}
+              {{ $row := slice "apache-arrow" "dask" "nuclio" "networkx" "numba" "scikit-learn" "xgboost" }}
               {{ range $row }}
-              {{ with site.GetPage (printf "_partners/%s" . ) }}
-              {{ $sm_logo := .Resources.Get "logo_400x.png" }}
-              {{ $default_logo := .Resources.Get "logo.png" }}
-              {{ $resized_image := ($sm_logo | default $default_logo).Resize "100x" }}
-              <a target="_blank" href="{{ .Params.github_repo_url | default .Params.github_org_url | default .Params.homepage_url }}">
-                <img src="{{ $resized_image.RelPermalink }}" alt="{{ .Params.name }} Logo">
-              </a>
-              {{ end }}
+                {{ with site.GetPage (printf "_partners/%s" . ) }}
+                  {{ $sm_logo := .Resources.Get "logo_400x.png" }}
+                  {{ $default_logo := .Resources.Get "logo.png" }}
+
+                  <!-- check to render logo differently. NetworkX logo is too small normally -->
```

Comment:
Can we get a logo that is the appropriate size instead of adding conditional logic here?

Reply:
So all the logos live inside `content/_partners/<partner_name>` in their high-resolution format. However, `layouts/index.html` will resize any image to a set width value (100px) and the NetworkX logo has a different aspect ratio than the other logos making it appear a lot smaller. 

I'm not sure what another workaround would be without modifying too much of the existing code